src/main/java/com/sihenzhang/crockpot/mixin/PiglinTasksMixin.java

```diff
@@ -99,9 +99,8 @@ private static void pickUpItemHandler(PiglinEntity piglinEntity, ItemEntity item
     @Inject(
             method = "pickUpItem(Lnet/minecraft/entity/monster/piglin/PiglinEntity;Lnet/minecraft/entity/item/ItemEntity;)V",
             at = @At(
-                    value = "JUMP",
-                    ordinal = 1,
-                    opcode = 154
+                    value = "INVOKE",
+                    target="Lnet/minecraft/entity/monster/piglin/PiglinTasks;putInInventory(Lnet/minecraft/entity/monster/piglin/PiglinEntity;Lnet/minecraft/item/ItemStack;)V"
```

Comment:
Setting the injection point to `INVOKE_AFTER` `equipItemIfPossible` will be the same as the original injection point.

Reply:
Also please set `ordinal`.

Reply:
Minecraft code:
```
boolean flag = p_234470_0_.equipItemIfPossible(itemstack);
      if (!flag){
>>injection point
        putInInventory(p_234470_0_, itemstack); 
}
```
flag is canBeEquiped.
So this is equivalent to original code, with lesser performance cost.

Reply:
Yes, you're right. So you can remove `!canBeEquipped` in the injection code because it's duplicated.